src/libs/actions/IOU.js

```diff
@@ -3723,6 +3715,30 @@ function getIOUReportID(iou, route) {
     return lodashGet(route, 'params.reportID') || lodashGet(iou, 'participants.0.reportID', '');
 }
 
+/**
+ * @param {Object} transaction
+ * @param {String} comment
+ * @param {Boolean} isDraft
+ * @param {String} iouType
+ * @param {String} reportID
+ */
+function setMoneyRequestDescription_temporaryForRefactor(transaction, comment, isDraft, iouType, reportID) {
+    // Only update comment if it has changed
+    if (comment.trim() === lodashGet(transaction, 'comment.comment', '')) {
+        return;
+    }
```

Comment:
Remove this check. If we need to compare with the current comment we should do so before calling `setMoneyRequestDescription_temporaryForRefactor`